src/platform/datapath_winuser.c

```diff
@@ -3273,25 +3734,82 @@ CxPlatDataPathUdpRecvComplete(
     }
 
 Drop:
-    //
-    // Try to start a new receive.
-    //
-    int32_t RetryCount = 0;
-    QUIC_STATUS Status;
-    do {
-        Status = CxPlatSocketStartReceive(SocketProc);
-    } while (!QUIC_SUCCEEDED(Status) && ++RetryCount < 10);
+}
```

Comment:
```suggestion
    ;
}
```

Reply:
But actually, why'd you remove this? Just unnecessary with RIO?

Reply:
This diff is looking a bit weird to me, but the reason this is moved is because we want to batch as many receive requests as possible, so we separate completing receives from posting new receives. We can then make a single syscall to submit all the receives we need.